[2023-02-07T09:39:45.987+0100] {__init__.py:50} DEBUG - Loading core task runner: StandardTaskRunner
[2023-02-07T09:39:45.999+0100] {taskinstance.py:1017} DEBUG - previous_execution_date was called
[2023-02-07T09:39:46.000+0100] {base_task_runner.py:69} DEBUG - Planning to run as the  user
[2023-02-07T09:39:46.003+0100] {taskinstance.py:769} DEBUG - Refreshing TaskInstance <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]> from DB
[2023-02-07T09:39:46.011+0100] {taskinstance.py:1093} DEBUG - <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2023-02-07T09:39:46.012+0100] {taskinstance.py:1093} DEBUG - <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2023-02-07T09:39:46.016+0100] {taskinstance.py:1093} DEBUG - <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2023-02-07T09:39:46.016+0100] {taskinstance.py:1093} DEBUG - <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]> dependency 'Trigger Rule' PASSED: True, The task instance did not have any upstream tasks.
[2023-02-07T09:39:46.016+0100] {taskinstance.py:1093} DEBUG - <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2023-02-07T09:39:46.016+0100] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]>
[2023-02-07T09:39:46.016+0100] {taskinstance.py:1093} DEBUG - <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2023-02-07T09:39:46.016+0100] {taskinstance.py:1093} DEBUG - <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]> dependency 'Trigger Rule' PASSED: True, The task instance did not have any upstream tasks.
[2023-02-07T09:39:46.016+0100] {taskinstance.py:1093} DEBUG - <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2023-02-07T09:39:46.017+0100] {taskinstance.py:1093} DEBUG - <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2023-02-07T09:39:46.022+0100] {taskinstance.py:1093} DEBUG - <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2023-02-07T09:39:46.022+0100] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [queued]>
[2023-02-07T09:39:46.023+0100] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-07T09:39:46.023+0100] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2023-02-07T09:39:46.023+0100] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-07T09:39:46.040+0100] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): get_hashtags> on 2023-02-07 08:38:33.165567+00:00
[2023-02-07T09:39:46.043+0100] {standard_task_runner.py:55} INFO - Started process 2412 to run task
[2023-02-07T09:39:46.045+0100] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'dagProcessusMetier', 'get_hashtags', 'manual__2023-02-07T08:38:33.165567+00:00', '--job-id', '2591', '--raw', '--subdir', 'DAGS_FOLDER/dagProcessusMetier.py', '--cfg-path', '/tmp/tmprfbhj9m3']
[2023-02-07T09:39:46.046+0100] {standard_task_runner.py:83} INFO - Job 2591: Subtask get_hashtags
[2023-02-07T09:39:46.046+0100] {cli_action_loggers.py:65} DEBUG - Calling callbacks: [<function default_action_log at 0x7fa663454790>]
[2023-02-07T09:39:46.058+0100] {settings.py:407} DEBUG - Disposing DB connection pool (PID 2412)
[2023-02-07T09:39:46.059+0100] {settings.py:266} DEBUG - Setting up DB connection pool (PID 2412)
[2023-02-07T09:39:46.059+0100] {settings.py:339} DEBUG - settings.prepare_engine_args(): Using NullPool
[2023-02-07T09:39:46.080+0100] {taskinstance.py:1017} DEBUG - previous_execution_date was called
[2023-02-07T09:39:46.082+0100] {task_command.py:388} INFO - Running <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [running]> on host TR.
[2023-02-07T09:39:46.083+0100] {taskinstance.py:769} DEBUG - Refreshing TaskInstance <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [running]> from DB
[2023-02-07T09:39:46.103+0100] {taskinstance.py:1017} DEBUG - previous_execution_date was called
[2023-02-07T09:39:46.104+0100] {taskinstance.py:848} DEBUG - Clearing XCom data
[2023-02-07T09:39:46.147+0100] {retries.py:80} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2023-02-07T09:39:46.153+0100] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=thomas
AIRFLOW_CTX_DAG_ID=dagProcessusMetier
AIRFLOW_CTX_TASK_ID=get_hashtags
AIRFLOW_CTX_EXECUTION_DATE=2023-02-07T08:38:33.165567+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-07T08:38:33.165567+00:00
[2023-02-07T09:39:46.153+0100] {__init__.py:117} DEBUG - Preparing lineage inlets and outlets
[2023-02-07T09:39:46.154+0100] {__init__.py:155} DEBUG - inlets: [], outlets: []
[2023-02-07T09:39:46.198+0100] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/tr/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/tr/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/tr/airflow/dags/dagProcessusMetier.py", line 114, in get_hashtags
    twt_id = kwargs['twt_id']
KeyError: 'twt_id'
[2023-02-07T09:39:46.198+0100] {taskinstance.py:769} DEBUG - Refreshing TaskInstance <TaskInstance: dagProcessusMetier.get_hashtags manual__2023-02-07T08:38:33.165567+00:00 [running]> from DB
[2023-02-07T09:39:46.202+0100] {taskinstance.py:2241} DEBUG - Task Duration set to 0.185748
[2023-02-07T09:39:46.203+0100] {taskinstance.py:1329} DEBUG - Clearing next_method and next_kwargs.
[2023-02-07T09:39:46.203+0100] {taskinstance.py:1318} INFO - Marking task as UP_FOR_RETRY. dag_id=dagProcessusMetier, task_id=get_hashtags, execution_date=20230207T083833, start_date=20230207T083946, end_date=20230207T083946
[2023-02-07T09:39:46.219+0100] {cli_action_loggers.py:83} DEBUG - Calling callbacks: []
[2023-02-07T09:39:46.219+0100] {standard_task_runner.py:100} ERROR - Failed to execute job 2591 for task get_hashtags ('twt_id'; 2412)
[2023-02-07T09:39:46.259+0100] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-02-07T09:39:46.272+0100] {dagrun.py:688} DEBUG - number of tis tasks for <DagRun dagProcessusMetier @ 2023-02-07 08:38:33.165567+00:00: manual__2023-02-07T08:38:33.165567+00:00, state:running, queued_at: 2023-02-07 08:38:33.178171+00:00. externally triggered: True>: 0 task(s)
[2023-02-07T09:39:46.272+0100] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
